lib/Backend/Encoder.cpp

```diff
@@ -855,7 +855,7 @@ void Encoder::TryCopyAndAddRelocRecordsForSwitchJumpTableEntries(BYTE *codeStart
 
 uint32 Encoder::GetCurrentOffset() const
 {
-    Assert(m_pc - m_encodeBuffer <= UINT_MAX);      // encode buffer size is uint32
+    AssertOrFailFast(m_pc - m_encodeBuffer <= UINT_MAX);      // encode buffer size is uint32
```

Comment:
>AssertOrFailFast [](start = 4, length = 16)

m_encodeBufferSize is a UINT32, so this failfast is redundant.  Given that it can get called multiple times per instr, let's leave it as an assert.